docs/help-topics/rhtap-import-configure-component/rhtap-import-configure-component.yml

```diff
@@ -11,18 +11,23 @@
     
     Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too.
 
-    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL.
-    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest appropriate deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
+    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL. Save any changes to the name by clicking the checkmark. Discard the changes and exit editing mode by clicking the X.
+    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest suitable deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
```

Comment:
Similar comment to as https://github.com/RedHatInsights/quickstarts/pull/85/files#r1214373699

We only recommend at most a single option. But we always provide a full list of options. But not all of those options are considered suitable. The wrong option will fail to build.

Reply:
@christianvogt Everything you're saying is correct, but is your concern about the "suitable deployment options" text here? There is at most one recommended runtime, but changing the runtime will change the deployment configuration "options", which this sentence is describing, with different CPU/Memory values. 

@missmesss The user can change the runtime, but if they change it from the recommended runtime, it may impact RHTAP's ability to successfully build or deploy their component (if we correctly detected it, and they change it to something wrong, it's bad, if we incorrectly identified their component's runtime, and they fix it, that's actually good). Changing the runtime can change the suggested deployment options, but they can also change those suggestions manually, which is covered in the CPU/Memory/Instances section here. If they change those values to unreasonable requests, it could cause problems - their request may be changed to the max the cluster allows, or their pipeline could get stuck, unable to be allocated the resources requested. Later on, if users have to pay for their resource consumption, setting those values higher would increase the cost of deployments, but that's not a concern yet.